Cargo.toml

```diff
@@ -46,6 +50,8 @@ serde_json = "1"
 sha2 = "0.10.1"
 thiserror = "1.0.30"
 url = "2.2.2"
+crc32fast = "1"
+xxhash-rust = {version = "0.8", features = ["xxh3"]}
```

Comment:
In Deno we use `twox-hash`. This seems to have a BSL license.

Reply:
Changed. To my surprise, twox-hash's implementation of xxhash64 is faster than xxhash-rust's xxh3!

Reply:
Added twox-hash as xxhash feature, and kept xxhash-rust as xxhash3 feature. 

I've confirmed twox-hash's xxhash3 implementation is outdated and not producing the same hash as xxhash-rust's. In my machine twox-hash's xxhash64 is faster than xxhash-rust's xxhash3, but different infrastructure might have different results. For this reason, I suggest to keep both.